Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/PodcastDataManager.swift

```diff
@@ -415,6 +428,18 @@ class PodcastDataManager {
     }
 
     func saveAutoAddToUpNextForAllPodcasts(autoAddToUpNext: Int32, dbQueue: FMDatabaseQueue) {
+        if FeatureFlag.settingsSync.enabled {
+            if let setting = AutoAddToUpNextSetting(rawValue: autoAddToUpNext) {
+                let podcasts = DataManager.sharedManager.allPodcasts(includeUnsubscribed: false)
+                // Could be replaced with SQLite JSON operation in iOS 16+
```

Comment:
Can we change here to use `json_extract`?

Reply:
👍 Updated to use `json_patch`